common/common-javassist/src/test/java/io/servicecomb/common/javassist/TestJavassistUtils.java

```diff
@@ -198,10 +191,8 @@ public void testGetNameForGenerateCode() {
 
   @Test
   public void managerClassPool() {
-    ClassLoader classLoader1 = new ClassLoader() {
-    };
-    ClassLoader classLoader2 = new ClassLoader() {
-    };
+    ClassLoader classLoader1 = new ClassLoader() { };
```

Comment:
after format code, it will rollback to be the deleted style.

Reply:
OK. fair enough. 
It is not a big issue, but I do not understand why we need to a line break if there is nothing inside.

Reply:
agree.
but it seems that hard to write this rule both in IntelliJ and eclipse

Reply:
Leave it as an issue and put it in the queue. :)